web/src/message_edit.js

```diff
@@ -556,17 +556,12 @@ function edit_message($row, raw_content) {
         $message_edit_content.val("");
         $message_edit_content.val(contents);
     }
-
-    // Scroll to keep the top of the message content text in the same
-    // place visually, adjusting for border and padding.
-    const edit_top = $message_edit_content[0].getBoundingClientRect().top;
-    const scroll_by = edit_top - content_top + 5 - 14;
-
-    edit_obj.scrolled_by = scroll_by;
-    message_viewport.scrollTop(message_viewport.scrollTop() + scroll_by);
```

Comment:
I would have written something similar to restore the scroll position but it doesn't seem like we need it. Also, looks like this was causing the message list to scroll when we didn't need to. LIkely these values are outdated too.

Reply:
Yeah, it seems to just work; maybe this was a quirk of the pre-grid design that we needed this.